source/common/config/grpc_stream.h

```diff
@@ -165,9 +148,8 @@ class GrpcStream : public Grpc::TypedAsyncStreamCallbacks<ResponseProto>,
   TokenBucketPtr limit_request_;
   const bool rate_limiting_enabled_;
   Event::TimerPtr drain_request_timer_;
-  // A queue to store requests while rate limited. Note that when requests cannot be sent due to the
-  // gRPC stream being down, this queue does not store them; rather, they are simply dropped.
-  std::queue<RequestQueueItem> request_queue_;
+
+  std::function<void(std::unique_ptr<ResponseProto>&&)> handle_response_cb_;
```

Comment:
Another candidate for a typedef/using or FooCallback class.

Reply:
Done (class).